exercises/input.js

```diff
@@ -49,14 +58,45 @@ class Input{
 		also needs to test the range, but note that the range is null by default
 		if it is still null, no range has been set, so don't test it
 			if it is not null, then test the range */
-	test(){
+    test(){
 
-	}
+        var successObject={};
+
+        var inputVal = this.testInput.val();
+
+        if (this.maxRange !== null && this.minRange !== null){
+            if(inputVal >= this.maxRange || inputVal <= this.minRange){
```

Comment:
technically the range would include the value, so being equal to the max is fine.  or equal to the min